src/BuiltinExtensions/ComfyUIBackend/ExtraNodes/SwarmComfyExtra/SwarmYolo.py

```diff
@@ -57,6 +69,32 @@ def seg(self, image, model_name, index):
             masks = masks[sortedindices]
```

Comment:
the sorting code in this function appears to be unmodified? Why is your code not editing the one thing it's meant to edit?

Reply:
The idea here is that if you used sorting segment, in this current implemention, it ignores the index, and return the first mask after the sort. So the largest (or smallest), top-most (bottom-most), etc. If no sorting is used, then the normal behavior as it used to be is used. I was focusing on @FurkanGozukara usecase for testing this sorting logic, but it wouldn't be hard to make it so you can sort, and also pass and index, so getting the second largest segment, for example.

Reply:
> The idea here is that if you used sorting segment, in this current implemention, it ignores the index, and return the first mask after the sort. So the largest (or smallest), top-most (bottom-most), etc. If no sorting is used, then the normal behavior as it used to be is used. I was focusing on @FurkanGozukara usecase for testing this sorting logic, but it wouldn't be hard to make it so you can sort, and also pass and index, so getting the second largest segment, for example.

i think this would be best case so people can both inpaint with any index with any prompt and still can sort by largest

Reply:
There's literally existing sort logic here, just. Replace it with customizable sorting. Furkan's case is already covered, it just sorts in the wrong order for what he wants